frontend/src/util/worksheet_utils.js

```diff
@@ -20,14 +16,17 @@ export function renderDuration(s) {
 
     s -= m * 60;
     var h = Math.floor(m / 60);
+
```

Comment:
Why add extra newlines?

Reply:
Removed those, they were added when I was debugging the logic.